pkg/backup/backup.go

```diff
@@ -292,6 +300,25 @@ func (kb *kubernetesBackupper) Backup(logger logrus.FieldLogger, backupRequest *
 	return err
 }
 
+func (kb *kubernetesBackupper) writeBackupVersion(tw *tar.Writer) error {
+	versionString := strconv.Itoa(BackupVersion)
+
+	hdr := &tar.Header{
+		Name:     "backup-version",
+		Size:     int64(len(versionString)),
+		Typeflag: tar.TypeReg,
+		Mode:     0644,
+		ModTime:  time.Now(),
+	}
+	if err := tw.WriteHeader(hdr); err != nil {
+		return err
+	}
+	if _, err := tw.Write([]byte(versionString)); err != nil {
```

Comment:
Do you want to include a newline? `fmt.Fprintln(tw, versionString)`, or even `fmt.Fprintf(tw, "%d\n", BackupVersion)`?